docs/physics/index.rst

```diff
@@ -0,0 +1,28 @@
+=========
+pace-physics
+=========
+
+pace-physics includes the Python implementation of GFS physics built using the GT4Py domain-specific language.
+Currently, only GFDL cloud microphysics is integrated into Pace.
+Additional physics schemes (NOAH land surface, GFS sea ice, scale-aware mass-flux shallow convection, hybrid eddy-diffusivity mass-flux PBL and free atmospheric turbulence, and rapid radiative transfer model)
+have been ported indendepently and are available in the `physics-standalone`_ repository,
+additional work is required to integrate these schemes.
+
+State container is highly recommended to keep track of physics variables
+because only certain variables interact with the rest of the model.
```

Comment:
```suggestion
State container is highly recommended to keep track of physics variables because only certain variables interact with the rest of the model.
```
This statement is confusing to me. Could you reference a `:py:class:` directly? Or if it's not possible to make it clear what exactly you're recommending users/devs do, can we remove this?

Reply:
Removed